person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './corrector'
 
 class Person
```

Comment:
- [ ] We should also be able to add a rental from the **Person** class. Kindly create a method (can be named `add_rental`) that takes in `book` and `date` as its parameters. The method should return an instance of **`Rental`** with `date`, `book`, and `self` (person) as its parameters